src/test/java/com/wikia/webdriver/testcases/adstests/TestUrlBuilder.java

```diff
@@ -12,151 +14,175 @@
+          "http://preview.de.naruto.wikia.com/wiki/Narutopedia",
+          "http://sandbox.de.naruto.wikia.com/wiki/Narutopedia",
+          "http://sandbox-mercurydev.de.naruto.wikia.com/wiki/Narutopedia",
+          "http://de.naruto.dmytror.wikia-dev.com/wiki/Narutopedia"
+      },
+      {
+          "de.gameofthrones", "Staffel_4", "http://de.gameofthrones.wikia.com/wiki/Staffel_4",
+          "http://preview.de.gameofthrones.wikia.com/wiki/Staffel_4",
+          "http://sandbox.de.gameofthrones.wikia.com/wiki/Staffel_4",
+          "http://sandbox-mercurydev.de.gameofthrones.wikia.com/wiki/Staffel_4",
+          "http://de.gameofthrones.dmytror.wikia-dev.com/wiki/Staffel_4"
+      },
+      {
+          "de.gta", "Fahrzeuge_(V)", "http://de.gta.wikia.com/wiki/Fahrzeuge_(V)",
+          "http://preview.de.gta.wikia.com/wiki/Fahrzeuge_(V)",
+          "http://sandbox.de.gta.wikia.com/wiki/Fahrzeuge_(V)",
+          "http://sandbox-mercurydev.de.gta.wikia.com/wiki/Fahrzeuge_(V)",
+          "http://de.gta.dmytror.wikia-dev.com/wiki/Fahrzeuge_(V)"
+      },
+      {
+          "de.fahrrad", "Reifenumfang_%28Tabelle%29",
+          "http://de.fahrrad.wikia.com/wiki/Reifenumfang_%28Tabelle%29",
+          "http://preview.de.fahrrad.wikia.com/wiki/Reifenumfang_%28Tabelle%29",
+          "http://sandbox.de.fahrrad.wikia.com/wiki/Reifenumfang_%28Tabelle%29",
+          "http://sandbox-mercurydev.de.fahrrad.wikia.com/wiki/Reifenumfang_%28Tabelle%29",
+          "http://de.fahrrad.dmytror.wikia-dev.com/wiki/Reifenumfang_%28Tabelle%29"
+      },
+      {
+          "de.bindingofisaac", "Items", "http://de.bindingofisaac.wikia.com/wiki/Items",
+          "http://preview.de.bindingofisaac.wikia.com/wiki/Items",
+          "http://sandbox.de.bindingofisaac.wikia.com/wiki/Items",
+          "http://sandbox-mercurydev.de.bindingofisaac.wikia.com/wiki/Items",
+          "http://de.bindingofisaac.dmytror.wikia-dev.com/wiki/Items"
+      },
+      {
+          "de.videospielehub", "Videospiele_Hub",
+          "http://de.videospielehub.wikia.com/wiki/Videospiele_Hub",
+          "http://preview.de.videospielehub.wikia.com/wiki/Videospiele_Hub",
+          "http://sandbox.de.videospielehub.wikia.com/wiki/Videospiele_Hub",
+          "http://sandbox-mercurydev.de.videospielehub.wikia.com/wiki/Videospiele_Hub",
+          "http://de.videospielehub.dmytror.wikia-dev.com/wiki/Videospiele_Hub"
+      },
+      {
+          "jedipedia", "Jedipedia:Hauptseite",
+          "http://jedipedia.wikia.com/wiki/Jedipedia:Hauptseite",
+          "http://preview.jedipedia.wikia.com/wiki/Jedipedia:Hauptseite",
+          "http://sandbox.jedipedia.wikia.com/wiki/Jedipedia:Hauptseite",
+          "http://sandbox-mercurydev.jedipedia.wikia.com/wiki/Jedipedia:Hauptseite",
+          "http://jedipedia.dmytror.wikia-dev.com/wiki/Jedipedia:Hauptseite"
+      },
+      {
+          "en.memory-alpha", "Portal:Main", "http://en.memory-alpha.wikia.com/wiki/Portal:Main",
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Portal:Main" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)